File: Learn the Basic/Pattern_printing/Increasing_Letter_Triangle_Pattern.py
def Increasing_Letter_Triangle_Pattern(n):
    for i in range(1,n+1):
        for j in range(ord('A'),ord('A')+i): # ord convert into its ascii 
            print(chr(j),end="") # ascii convert into its character value
        print()
n=int(input("Enter the value of n: "))
Increasing_Letter_Triangle_Pattern(n)

File: Learn the Basic/Pattern_printing/Increasing_Letter_Triangle_Pattern.cpp
#include<iostream>
using namespace std;
void Increasing_Letter_Triangle_Pattern(int n){
    for(int i=1;i<=n;i++){
        for(char j='A';j<'A'+i;j++){
            cout<<j;
        }
        cout<<endl;
    }
}
int main(){
    int n;
    cout<<"Enter the value of n: ";
    cin>>n;
    Increasing_Letter_Triangle_Pattern(n);
    return 0;
}